packages/zowe-explorer-ftp-extension/__tests__/__unit__/Uss/ZoweExplorerFtpUssApi.unit.test.ts

```diff
@@ -89,7 +89,12 @@ describe("FtpUssApi", () => {
                 file: localFile,
             },
         };
-        const result = await UssApi.putContents(mockParams.inputFilePath, mockParams.ussFilePath, undefined, undefined, "test", true);
+        const result = await UssApi.putContent(mockParams.inputFilePath, mockParams.ussFilePath, {
+            binary: undefined,
+            localEncoding: undefined,
```

Comment:
I know this is just test code, but we can probably omit these or update the type in the API so that `binary` and `localEncoding` are optional parameters for the object.